### PR TITLE
fix(typeahead): block user to associate area/iteration from detail panel

### PR DESCRIPTION
**Pull request by vikram-raj**
Fix: https://github.com/fabric8io/fabric8-planner/issues/1847

### CODE REVIEW

**Comment by fabric8cd** [comment]
@vikram-raj snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1963-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@vikram-raj snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1963-fabric8-planner.int.rdu2c.fabric8.io

File: src/app/components/typeahead-dropdown/typeahead-dropdown.component.ts [hunk 1]
```diff
@@ -169,4 +194,14 @@ export class TypeaheadDropdown implements OnInit, OnChanges {
       this.sortValuesByLength(this.filteredValues);
     }
   }
+
+  listenToEvent() {
+    this.eventListeners.push(
+      this.broadcaster.on<string>('logout')
+        .subscribe(message => {
+          this.loggedIn = false;
+          console.log('viviv');
```
Comment (nimishamukherjee) [inline src/app/components/typeahead-dropdown/typeahead-dropdown.component.ts]: Remove console log

**Comment by fabric8cd** [comment]
@vikram-raj snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1963-fabric8-planner.int.rdu2c.fabric8.io

**Pull request merged**